src/game/default/g_entity.c

```diff
@@ -215,6 +280,7 @@ static const g_field_t fields[] = {
 	{ "gameplay", SOFS(gameplay), F_STRING, FFL_SPAWN_TEMP },
 	{ "hook", SOFS(hook), F_STRING, FFL_SPAWN_TEMP },
 	{ "teams", SOFS(teams), F_STRING, FFL_SPAWN_TEMP },
+	{ "num_teams", SOFS(num_teams), F_STRING, FFL_SPAWN_TEMP },
```

Comment:
Oh, and we should make sure that the `maps.list` parser supports multi-team support too. That way, we can add 3-team CTF maps to a server and specify `teams 3` there.